src/Tasks/Microsoft.CSharp.CurrentVersion.targets

```diff
@@ -234,92 +234,80 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
        <!-- Condition is to filter out the _CoreCompileResourceInputs so that it doesn't pass in culture resources to the compiler -->
        <!-- Don't run analyzers for Csc task on XamlPrecompile pass, we only want to run them on core compile. -->
-    <Csc Condition="'%(_CoreCompileResourceInputs.WithCulture)' != 'true'"
-         AdditionalLibPaths="$(AdditionalLibPaths)"
-         AddModules="@(AddModules)"
-         AdditionalFiles="@(AdditionalFiles)"
-         AllowUnsafeBlocks="$(AllowUnsafeBlocks)"
-         AnalyzerConfigFiles="@(EditorConfigFiles)"
-         Analyzers="@(Analyzer)"
-         ApplicationConfiguration="$(AppConfigForCompiler)"
-         BaseAddress="$(BaseAddress)"
-         CheckForOverflowUnderflow="$(CheckForOverflowUnderflow)"
-         ChecksumAlgorithm="$(ChecksumAlgorithm)"
-         CodeAnalysisRuleSet="$(ResolvedCodeAnalysisRuleSet)"
-         CodePage="$(CodePage)"
-         DebugType="$(DebugType)"
-         DefineConstants="$(DefineConstants)"
-         DelaySign="$(DelaySign)"
-         DisabledWarnings="$(NoWarn)"
-         DisableSdkPath="$(DisableSdkPath)"
-         DocumentationFile="@(DocFileItem)"
-         EmbedAllSources="$(EmbedAllSources)"
-         EmbeddedFiles="@(EmbeddedFiles)"
-         EmitDebugInformation="$(DebugSymbols)"
-         EnvironmentVariables="$(CscEnvironment)"
-         ErrorEndLocation="$(ErrorEndLocation)"
-         ErrorLog="$(ErrorLog)"
-         ErrorReport="$(ErrorReport)"
-         Features="$(Features)"
-         InterceptorsPreviewNamespaces="$(InterceptorsPreviewNamespaces)"
-         FileAlignment="$(FileAlignment)"
-         GeneratedFilesOutputPath="$(CompilerGeneratedFilesOutputPath)"
-         GenerateFullPaths="$(GenerateFullPaths)"
-         HighEntropyVA="$(HighEntropyVA)"
-         Instrument="$(Instrument)"
-         KeyContainer="$(KeyContainerName)"
-         KeyFile="$(KeyOriginatorFile)"
-         LangVersion="$(LangVersion)"
-         LinkResources="@(LinkResource)"
-         MainEntryPoint="$(StartupObject)"
-         ModuleAssemblyName="$(ModuleAssemblyName)"
-         NoConfig="true"
-         NoLogo="$(NoLogo)"
-         NoStandardLib="$(NoCompilerStandardLib)"
-         NoWin32Manifest="$(NoWin32Manifest)"
-         Nullable="$(Nullable)"
-         Optimize="$(Optimize)"
-         Deterministic="$(Deterministic)"
-         PublicSign="$(PublicSign)"
-         OutputAssembly="@(IntermediateAssembly)"
-         OutputRefAssembly="@(IntermediateRefAssembly)"
-         PdbFile="$(PdbFile)"
-         Platform="$(PlatformTarget)"
-         Prefer32Bit="$(Prefer32Bit)"
-         PreferredUILang="$(PreferredUILang)"
-         ProjectName="$(MSBuildProjectName)"
-         ProvideCommandLineArgs="$(ProvideCommandLineArgs)"
-         References="@(ReferencePathWithRefAssemblies)"
```

Comment:
I don't know why this was using the old `@(ReferencePath)` before. Scared to change it but filed https://github.com/dotnet/msbuild/issues/9787.